mixlib-log.gemspec

```diff
@@ -13,9 +13,10 @@ Gem::Specification.new do |gem|
   gem.has_rdoc = true
   gem.extra_rdoc_files = ["README.md", "LICENSE", "NOTICE"]
   gem.files = Dir["lib/**/*"] + Dir["spec/**/*"] + ["Gemfile", "Rakefile", ".gemtest", "mixlib-log.gemspec"]
+  gem.required_ruby_version = ">= 2.2"
   gem.add_development_dependency "rake"
   gem.add_development_dependency "rspec", "~> 3.4"
-  gem.add_development_dependency "chefstyle", "~> 0.3"
+  gem.add_development_dependency "chefstyle", "= 0.4.0"
```

Comment:
i kind of think this dep could just float on the latest release; it's not like these gems get a lot of changes.